service/controller/crossplane/crossplane.go

```diff
@@ -0,0 +1,66 @@
+package crossplane
+
+import (
+	"github.com/giantswarm/k8sclient/v7/pkg/k8sclient"
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/micrologger"
+	"github.com/giantswarm/operatorkit/v7/pkg/controller"
+	"github.com/giantswarm/operatorkit/v7/pkg/resource"
+	rbacv1 "k8s.io/api/rbac/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	"github.com/giantswarm/rbac-operator/pkg/project"
+	"github.com/giantswarm/rbac-operator/service/internal/accessgroup"
+)
+
+type CrossplaneConfig struct {
+	K8sClient k8sclient.Interface
+	Logger    micrologger.Logger
+
+	CustomerAdminGroups []accessgroup.AccessGroup
+}
+
+type Crossplane struct {
+	*controller.Controller
+}
+
+func NewCrossplane(config CrossplaneConfig) (*Crossplane, error) {
+	var err error
+
+	var resources []resource.Interface
+	{
+		c := crossplaneResourcesConfig(config)
+
+		resources, err = newCrossplaneResources(c)
+		if err != nil {
+			return nil, microerror.Mask(err)
+		}
+	}
+
+	var clusterRoleAuthController *controller.Controller
+	{
+		c := controller.Config{
+			K8sClient: config.K8sClient,
+			Logger:    config.Logger,
+			NewRuntimeObjectFunc: func() client.Object {
```

Comment:
Does Crossplane's RBAC Manager use any label which could be used here for selecting resources, to narrow down the scope?

Reply:
They do, I was just afraid that if they change the labels at any point, and we won't notice, the binding will stop working. The same applies to ClusterRole name, but I expect this to not happen that easily. Do you think we have enough ClusterRoles for that to become a problem?

We can use the following labels:

```
  labels:
    app: crossplane
    app.kubernetes.io/component: cloud-infrastructure-controller
    app.kubernetes.io/instance: crossplane-crossplane
    app.kubernetes.io/managed-by: Helm
    app.kubernetes.io/name: crossplane
    app.kubernetes.io/part-of: crossplane
```